### PR TITLE
add intl selector

### PR DESCRIPTION
**Pull request by jerem1508**

### CODE REVIEW

File: client/src/layout/Layout.tsx [hunk 1]
```diff
@@ -1,21 +1,79 @@
+import { useEffect } from 'react';
+import { Outlet, useSearchParams } from 'react-router-dom';
 import { Button, Header, Logo, Service, FastAccess } from '@dataesr/dsfr-plus';
```
Comment (annelhote) [inline client/src/layout/Layout.tsx:3]: Sort import

File: client/src/layout/Layout.tsx [hunk 2]
```diff
@@ -1,21 +1,79 @@
...
-          <Button className="fr-btn fr-icon-theme-fill" aria-controls="fr-theme-modal" data-fr-opened="false">
-            Changer de thème
+          <Button
+            as="a"
+            href="/"
+            icon="github-fill"
+            size="sm"
+            variant="text"
+          >
+            {(searchParams.get('language') === 'FR') ? "Explorer d'autres tableaux de bord" : "Explore dashboards"}
```
Comment (annelhote) [inline client/src/layout/Layout.tsx:35]: It would be cleaner to have dedicated translation files.

File: client/src/router.tsx [hunk 3]
```diff
@@ -13,13 +12,11 @@ export default function Router() {
...
     <Routes>
       <Route path="/" element={<HomePage />} />
-      <Route element={<Layout />}>
-        <Route path="/atlas/*" element={<AtlasRoutes />} />
-        <Route path="/european-projects/*" element={<EuropeanProjectsRoutes />} />
-        <Route path="/open-alex/*" element={<OpenAlexRoutes />} />
-        <Route path="/teds/*" element={<TedsRoutes />} />
-        <Route path="/finance-universite/*" element={< FinanceUniversityRoutes />} />
-      </Route>
+      <Route path="/atlas/*" element={<AtlasRoutes />} />
```
Comment (annelhote) [inline client/src/router.tsx:15]: I do not get the reason why you moved the Layout element from here to each page ?

**Review by annelhote — CHANGES_REQUESTED** [review]

File: client/src/router.tsx (hunk 3 above)
Comment (jerem1508) [reply client/src/router.tsx:15]: each sub site has to manage his translation

**Review by annelhote — APPROVED** [review]

**Pull request merged**